src/Razor/src/Microsoft.VisualStudio.Editor.Razor/Telemetry/TelemetryHelpers.cs

```diff
@@ -0,0 +1,93 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics;
+using Microsoft.VisualStudio.Telemetry;
+
+namespace Microsoft.AspNetCore.Razor.Telemetry;
```

Comment:
These were classes previously in `TelemetryReporter` that I refactored out so they could be used by both `TelemetryReporter` and `DevKitTelemetryReporter`